Gemfile.tt

```diff
@@ -14,15 +14,14 @@ gem "pgcli-rails"
 <%= gemfile_entry "puma" -%>
 gem "rack-canonical-host"
 gem "rails", "~> <%= Rails.version[/\d+\.\d+\.\d+/] %>"
-<%= gemfile_entry "redis", force: true -%>
-gem "sidekiq"
 <%= gemfile_entry "sprockets-rails" -%>
 <%= gemfile_entry "stimulus-rails" -%>
 <%= gemfile_entry "turbo-rails" -%>
 <%= %Q{gem "vite_rails"\n} if install_vite? -%>
 
 group :production do
   gem "postmark-rails"
+  gem "sidekiq"
```

Comment:
Sidekiq is only needed in production. In development and test, we can use the default "async" ActiveJob adapter.